jupyter-extension/jupyterlab_pachyderm/pps_client.py

```diff
@@ -81,6 +81,8 @@ def from_notebook(cls, notebook_path: Union[str, Path]) -> "PpsConfig":
         if input_spec_str is None:
             raise ValueError("field input_spec not set")
         input_spec_dict = yaml.safe_load(input_spec_str)
+        if input_spec_dict is None:
```

Comment:
You may be right here, I found this when I accidentally submitted a pipeline request without the input spec. Since this is a backport, I can update this message in a future PR.